subgraphs/makerdao/protocols/makerdao/config/templates/makerdao.template.yaml

```diff
@@ -50,6 +50,10 @@ dataSources:
         - event: LogNote(indexed bytes4,indexed bytes32,indexed bytes32,indexed bytes32,bytes)
           topic0: "0x7608870300000000000000000000000000000000000000000000000000000000"
           handler: handleVatFrob
+        ### CDP Confiscation
+        - event: LogNote(indexed bytes4,indexed bytes32,indexed bytes32,indexed bytes32,bytes)
```

Comment:
The fix itself is this part I'm assuming?

Reply:
Correct.